extensions/blocks/podcast-player/templates/podcast-header.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Podcast Header template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+/**
+ * Template variables.
+ *
+ * @var array  $template_props
+ * @var string $player_id
+ * @var string $title
+ * @var string $link
+ * @var array  $primary_colors
+ */
+
+/**
+ * Block attributes
+ */
+$attributes               = (array) $template_props['attributes']; // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$show_cover_art           = (bool) $attributes['showCoverArt'];
+$show_episode_description = (bool) $attributes['showEpisodeDescription'];
+
+// Current track.
+$track = ! empty( $template_props['tracks'] ) ? $template_props['tracks'][0] : array();
```

Comment:
Should we check if `$template_props['tracks']` is an array as well?

Reply:
sure!